clab/config.go

```diff
@@ -74,6 +74,7 @@ var kinds = []string{
 	"mysocketio",
 	"host",
 	"cvx",
+	"ixia-c-one",
```

Comment:
we are going to move to _fully qualified_ kind names, which will include vendor info
this means, that I'd like to ask you to change the kind to `keysight_ixia-c-one`